styles/homemaincontent.css

```diff
@@ -0,0 +1,244 @@
+body {
+    min-height: 100%;
+    font-family: "Poppins", sans-serif;
+    display: flex;
+    flex-direction: column;
+}
+  
+.container {
+    display: flex;
+    flex-grow: 1;
+}
+  
+.sidebar {
+    flex-basis: 25%;
+}
+  
+.main-content {
+    background-image: url("../images/pexels-c-reiners-907786.jpg") ;
```

Comment:
better to rename the jpg image name which is easy to find

Reply:
okay